frontend/src/App.js

```diff
@@ -1,26 +1,143 @@
+                <Card.Body>
+                  <Card.Title>Card Title</Card.Title>
+                  <Card.Text>
+                    Some quick example text to build on the card title and make up the bulk of
+                    the card's content.
+                  </Card.Text>
+                  <Button variant="princ">Details</Button>
+                  <Button variant="princ">Contact</Button>
+                </Card.Body>
+            </Card> <br/> <br/>
+
+            
+
+          <Form>
+              <Form.Group controlId="formControlInput">
+                      <Form.Label>Text Input: </Form.Label>
+                      <Form.Control type="text" placeholder="Placeholder" size="large" />
+              </Form.Group>
+              <Form.Group controlId="formControlTextArea">
+                      <Form.Label>TextArea: </Form.Label>
+                      <Form.Control as="textarea" rows="4" placeholder="Placeholder" size="large" />
+              </Form.Group>
+              <Form.Group controlId="formControlSelect">
+                      <Form.Label>Select:</Form.Label>
+                      <Form.Control as="select">
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                        <option>4</option>
+                        <option>5</option>
+                      </Form.Control>
+              </Form.Group>
+              <Form.Group controlId="formRange" className="formRange" >
+                      <Form.Label>Range:</Form.Label>
+                      <Form.Control type="range" custom/>
+              </Form.Group>
+              <Form.Group controlId="formCheckbox">
+                      <Form.Check type="checkbox" label="Check me out" />
+              </Form.Group>
+              <br />
+              <Form.Group controlId="formControlRadio">
+                  <Form.Label >
+                      Radios:
+                  </Form.Label>
+                        <Form.Check
+                            type="radio"
+                            label="first radio"
+                            name="formHorizontalRadios"
+                            id="formHorizontalRadios1"
+                        />
+                        <Form.Check
+                          type="radio"
+                          label="second radio"
+                          name="formHorizontalRadios"
+                          id="formHorizontalRadios2"
+                        />
+                        <Form.Check
+                          type="radio"
+                          label="third radio"
+                          name="formHorizontalRadios"
+                          id="formHorizontalRadios3"
+                        />
+                    
+              </Form.Group>
+           
+          </Form>
+          
+          <br /><br />
+
+        </div>
+
+      </div>
+
+    );
+  }
 }
 
 export default App;
```

Comment:
App needs to be a functional component (not class component in order for hooks & custom hooks to work). 

Refactor that Bootstrap styling either in index.css, or App.css or for every particular component (for example, a Button component, create a simple Component of your own, import Bootstrap button there and add a .css file for your own component and only import your own component).